controls/anssi.yml

```diff
@@ -372,8 +372,10 @@ controls:
     - accounts_passwords_pam_faillock_unlock_time
 
     # Do not reuse last two passwords
-    - var_password_pam_unix_remember=2
-    - accounts_password_pam_unix_remember
```

Comment:
@marcusburghardt Could you please remove the reference from the rule? Thanks

Reply:
Good question. Should we keep the benchmark reference only on rules which are actually included in the controlfile? Or any rule which would satisfy the requirement should have the reference? In this case, the `accounts_password_pam_unix_remember` rule is still valid for the `anssi: BP28(R18)`, regardless of our recommendation of other rules in the controlfile. Maybe keeping the reference could be helpful.

Reply:
Good point, I think it depends on the policy.

References to technical policies like STIG or CIS should probably be restricted to rules that align with the recommendations/requirements.

For ANSSI, which has high level and level requirements, it will depend on the Recommendation.
R18 establishes some high level and other low level recommendations. It is not defined there how the re-use of passwords should be defined.

We can keep the reference in the previously used rule.